Sejima/SejimaTests/MUOverlayTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  MUOverlayTests.swift
+//  Tests
+//
+//  Created by Loïc GRIFFIE on 14/11/2018.
+//  Copyright © 2018 Loïc GRIFFIE. All rights reserved.
+//
+
+import XCTest
+import UIKit
+@testable import Sejima
+
+class MUOverlayTests: XCTestCase {
+    func testDefaultValues() {
+        let overlay = MUOverlay()
+        XCTAssertNotNil(overlay)
+
+        XCTAssertEqual(overlay.style, .round)
+        XCTAssertEqual(overlay.borderWidth, 2)
+        XCTAssertEqual(overlay.radius, 0)
+        XCTAssertEqual(overlay.borderColor, .white)
+    }
+
+    func testCustomValues() {
+        let overlay = MUOverlay()
+        XCTAssertNotNil(overlay)
+
+        overlay.style = .custom(8)
```

Comment:
.custom(8) will be erase by radius = 4.
Should be nice to test it before or to add more tests in testOverlayRadius.